Tests/MVC6/Boilerplate.AspNetCore.TagHelpers.Test/project.json

```diff
@@ -1,20 +1,23 @@
 {
   "version": "1.0.0-*",
-
   "testRunner": "xunit",
-
+  "title": "",
+  "buildOptions": {
+    "debugType": "portable",
+    "preserveCompilationContext": true,
+    "additionalArguments": [ "/ruleset:../../../MinimumRecommendedRulesWithStyleCop.ruleset" ],
+    "copyToOutput": {"include": [ "xunit.runner.json" ]},
```

Comment:
For some reason I remember this being the solution to an issue I had when I initially created the test kits from a bug thread, I think you were apart of. I could have just gotten it from [Configuring xUnit.net with JSON](https://xunit.github.io/docs/configuring-with-json.html) but, seeing it in a thread sounds much more familiar.   None-the-less it seems to run fine now with out it.  I wrote these a while ago so if it was related to a bug it's been resolved.